test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -1498,12 +1498,27 @@ class test_ucp_wireup_keepalive : public test_ucp_wireup {
                                          UCP_FEATURE_RMA | UCP_FEATURE_TAG);
     }
 
+    ucp_ep_params_t get_ep_params() {
+        ucp_ep_params_t params;
+        memset(&params, 0, sizeof(params));
+        params.field_mask      = UCP_EP_PARAM_FIELD_ERR_HANDLING_MODE |
+                                 UCP_EP_PARAM_FIELD_ERR_HANDLER;
+        params.err_mode        = UCP_ERR_HANDLING_MODE_PEER;
+        params.err_handler.cb  = err_cb;
+        params.err_handler.arg = reinterpret_cast<void*>(this);
+        return params;
+    }
+
     void init() {
         test_ucp_wireup::init();
 
         sender().connect(&receiver(), get_ep_params());
         receiver().connect(&sender(), get_ep_params());
     }
+
+protected:
+    static void err_cb(void *arg, ucp_ep_h ep, ucs_status_t status) {
+    }
```

Comment:
seems like something missed...

Reply:
this callback is not called - used for initialization only

Reply:
then can be NULL

Reply:
done